### PR TITLE
Xshooter tuneup

### PR DESCRIPTION
**Pull request by jhennawi**
This PR accompanies the xshooter_tuneup branch. The main change is realizing some setups are reducing multiple science frames which is not necessary.

### CODE REVIEW

File: coadd1d_files/gemini_gnirs_32_sb_sxd.coadd1d [hunk 1]
```diff
@@ -6,6 +6,6 @@
 coadd1d read
   Science/spec1d_cN20170331S0216-pisco_GNIRS_20170331T085412.181.fits OBJ0652-DET01
   Science/spec1d_cN20170331S0217-pisco_GNIRS_20170331T085933.097.fits OBJ0205-DET01
-  Science/spec1d_cN20170331S0220-pisco_GNIRS_20170331T091538.731.fits OBJ0658-DET01
-  Science/spec1d_cN20170331S0221-pisco_GNIRS_20170331T092059.597.fits OBJ0212-DET01
+  Science/spec1d_cN20170331S0220-pisco_GNIRS_20170331T091538.731.fits OBJ0659-DET01
```
Comment (profxj) [inline coadd1d_files/gemini_gnirs_32_sb_sxd.coadd1d:9]: Can't we fix this annoyance for once and for all?

File: dev_algorithms/wavelengths/template_files/SOAR_Goodman/400_SYZY/README [hunk 2]
```diff
@@ -0,0 +1,2 @@
+MasterWaveCalib_A_1_01_M1.fits -- From the Dev suite data (old redux)
```
Comment (profxj) [inline dev_algorithms/wavelengths/template_files/SOAR_Goodman/400_SYZY/README:1]: would have thought this was merged in already.  

Maybe I need to submit a PR..

**Review by profxj — CHANGES_REQUESTED** [review]
One request

File: coadd1d_files/gemini_gnirs_32_sb_sxd.coadd1d (hunk 1 above)
Comment (jhennawi) [reply coadd1d_files/gemini_gnirs_32_sb_sxd.coadd1d:9]: I thought this was what pypeit_collate1d was for?

File: dev_algorithms/wavelengths/template_files/SOAR_Goodman/400_SYZY/README (hunk 2 above)
Comment (jhennawi) [reply dev_algorithms/wavelengths/template_files/SOAR_Goodman/400_SYZY/README:1]: Yes I was similarly confused by this.

**Review by profxj — APPROVED** [review]
merging

**Pull request merged**